plugins/editor/src/editor/Editor.cpp

```diff
@@ -18,6 +18,10 @@
 #include <absl/strings/ascii.h>
 #include <absl/strings/numbers.h>
 #include <ghc/fs_std.hpp>
+
+#define STB_IMAGE_IMPLEMENTATION
+#include <stb_image/stb_image.h>
```

Comment:
Compile stb_image implementation in a separate cpp file, to save compile time in Editor.
(example in `external/st_audiofile/src/st_audiofile_libs.c`)